.golangci.yml

```diff
@@ -17,7 +17,9 @@ severity:
 
 issues:
   # use these new lint checks on code since #2574
-  new-from-rev: eb019291beed556ec6ac1ceb4a15114ce4df0c57
+  #new-from-rev: eb019291beed556ec6ac1ceb4a15114ce4df0c57
+  # use stricter enforcement since ##4410
+  new-from-rev: fd488f806dcbc2586f585155eea0180c30287f70
```

Comment:
What is the new checkpoint, and how was it chosen?

Is there a convenient way to run all of the checks against a single file?  I'd occasionally like to clean up some older code, if I'm making a lot of changes in an area.

Reply:
It's the most recent PR. We have a number of `errcheck` and `ineffassign` failures since `eb019291beed556ec6ac1ceb4a15114ce4df0c57`.

There are a lot of options for `golangci-lint run`, but I don't see one to specifically override the rev. I think it would be good to work backwards over time, but that doesn't really help you target specific packages.

Reply:
Or, perhaps we can run it twice (I'm assuming the caching makes that fast), once with the old config on the old checkpoint, and once with more stringent checks on a more recent one.

It would be very cool if you could associate a rev with each checker, when you decide as a group to adopt a new policy.

Reply:
`--new-from-rev REV` seems to do it at the CLI

Reply:
If it's a common check, you could maintain additional config files and pass them in with -c. There is an exclude directory option, so you could filter that way.

There are really a huge number of options: https://github.com/golangci/golangci-lint/blob/master/.golangci.reference.yml

Reply:
that is https://github.com/algorand/go-algorand/pull/4241

Reply:
so review + merge https://github.com/algorand/go-algorand/pull/4241 and I think this does not need to change.

Reply:
I have always hoped we will move this commit pointer backwards, never forwards..

Reply:
Now that #4241 is merged, can you rebase onto it, and see if we can move leave the rev as is? (Or back it up?)

Reply:
@jannotti thanks for the heads up, I missed that. I reverted the `new-from-rev` change and all looks good.